.github/workflows/rust.yml

```diff
@@ -58,15 +58,17 @@ jobs:
           - 1.22.0
           - beta
           - stable
-    steps:      
-      - name: Checkout Crate 
+    steps:
+      - name: Checkout Crate
         uses: actions/checkout@v2
       - name: Checkout Toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: ${{ matrix.rust }}
           override: true
+      - name: Pin cc if rust 1.22
```

Comment:
Not sure I am getting why do we need this once we moved MSRV to 1.29

Reply:
It's not ?